src/main/java/mate/academy/springboot/web/controller/BookController.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.springboot.web.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.web.dto.BookDto;
+import mate.academy.springboot.web.dto.CreateBookRequestDto;
+import mate.academy.springboot.web.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to applicaiton.properties

```suggestion
@RequestMapping("/books")
```